From 60b07146e7629bc9fc6554893fd9611cacb2b496 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Mon, 16 Sep 2013 09:48:05 -0400
Subject: [PATCH 13/28] Coverity fix - 11952

Fix resource leak in linked-attributes fixup task
(cherry picked from commit e92d2165463afe7a7606af6d13532642fa3462c1)
(cherry picked from commit 682920001822c14b45c08e2c668615762ab02c3d)
---
 ldap/servers/plugins/linkedattrs/fixup_task.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/ldap/servers/plugins/linkedattrs/fixup_task.c b/ldap/servers/plugins/linkedattrs/fixup_task.c
index 537544d..ff6dd82 100644
--- a/ldap/servers/plugins/linkedattrs/fixup_task.c
+++ b/ldap/servers/plugins/linkedattrs/fixup_task.c
@@ -236,6 +236,7 @@ linked_attrs_fixup_links(struct configEntry *config)
         if (plugin_is_betxn) {
             Slapi_DN *fixup_dn = slapi_sdn_new_dn_byref(config->scope);
             Slapi_Backend *be = slapi_be_select(fixup_dn);
+            slapi_sdn_free(&fixup_dn);
 
             if (be) {
                 fixup_pb = slapi_pblock_new();
-- 
1.7.1

